applications/incompressible_navier_stokes/fda_benchmark/include/postprocessor.h

```diff
@@ -141,16 +141,19 @@ class PostProcessorFDA : public PostProcessor<dim, Number>
     }
 
     // evaluation of results along lines
-    if(line_plot_calculator_statistics->time_control_statistics.time_control.needs_evaluation(
-         time, time_step_number))
+    if(line_plot_calculator_statistics)
```

Comment:
what does this if-statement mean? `line_plot_calculator_statistics` is not a boolean variable, right?

Reply:
Yes, its the same as:
```
if(line_plot_calculator_statistics!=nullptr)
```